.circleci/config.yml

```diff
@@ -3,11 +3,11 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: "ubuntu-1604:201903-01"
+      image: "ubuntu-2004:202010-01"
```

Comment:
FMI how is this decision made to use one image or the other? Is there a requirement in the CNI docs?

Reply:
No it was only that the build image is pretty old, it's EOL since April IIRC.

Reply:
Yeah this is just a build an image for the docker image, not used in the final CNI image